src/calculatedBullsAndCows.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const calculatedBullsAndCows = (secret, guess) => {
```

Comment:
Please, use descriptive names
```suggestion
const calculatedBullsAndCows = (secretNumber, userNumber) => {
```

Reply:
Fix